applications/tokenomics_enhancement_neutroswap.md

```diff
@@ -0,0 +1,301 @@
+| 2.  | Documentation | Deliver comprehensive documentation for Neutroswap V2, encompassing a range of instructional materials, example files, and detailed guidelines within the project's repository.|
+
+### Milestone 1 - Smart Contract Development
+
+- **Estimated duration:** 4 weeks
+- **FTE:** 3
+- **Costs:** 25,000 USD
+
+| ID  | Deliverable                   | Specification                                                                                                                                                                                      |
+| --- | ----------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| 0a. | License                       | Apache 2.0                                                                                                                                                                                         |
+| 0b. | Documentation                 | We will provide both **inline documentation** of the code and a technical **tutorial** that explains how users can utilize all of the new updates works.                                           |
+| 0c. | Unit Test                     | Core functions will be fully covered by unit tests to ensure functionality and robustness. These test will be provided in the repository for the public |
+| 0d. | Testing Guide                 | The functions of this milestone can be tested by running the unit tests we provide, and guidance on how to run these tests is provided in the Readme file. |
+| 1.  | Core Contracts | Publish the all the necessary smart contract such as xNEUTRO, spNFTs, plugins in a public repo with the audit report included. This smart contract will also publicly available on our team Github |
+| 2.  | Tokenomics contracts | xNEUTRO can only be redeem with vesting period. It will allow burning mechanism for $NEUTRO. However, $NEUTRO can be converted into $xNEUTRO in instant. |
+| 3.  | Plugins contracts | Plugin contracts are enabling users to allocate their xNEUTRO into their desired plugins: Dividends, Yield boosters and launchpad |
+| 4.  | Protocol revenue contracts | Develop and implement smart contracts to enhance Neutroswap's protocol revenue generation capabilities and enable efficient distribution of rewards to $xNEUTRO holders. |
+
+### Milestone 2 - Graph Node Deployment, Subgraphs, and Web Development
+
+- **Estimated duration:** 4 weeks
+- **FTE:** 4
+- **Costs:** 15,000 USD
```

Comment:
The rate per week for these milestones (given the FTE count) varies wildly.  Is this reflective of the different skillsets involved in the milestones or is it accounting for the unknowable risks that the first milestone will discover as the later milestones are designed? Or is it something else entirely?

Reply:
Yes exactly. To able to deliver the milestone we need vary skillsets, hence different rate. In the milestone 2, we request more budget because of the rate of smart contract developers are higher than designer, frontend, non-web3 backend engineer in milestone 3.

Reply:
Is there anything we can add?